ntoml/netlify_toml.go

```diff
@@ -148,12 +115,6 @@ func LoadFrom(paths ...string) (*NetlifyToml, error) {
 			switch extension {
 			case ".toml":
```

Comment:
could you maybe simplify this code since we don't really care about different extensions anymore?

Reply:
done 👍